src/scripts/update_funding_round.js

```diff
@@ -111,26 +111,80 @@ const main = async () => {
 
-            const maxGrantOCEAN = maxGrantUSD / tokenPrice
-            const earmarkedOCEAN = earmarkedUSD / tokenPrice
-            const fundingAvailableOCEAN = fundingAvailableUSD / tokenPrice
+            let maxGrant = 0
+            let earmarked = 0
+            let fundingAvailable = 0
 
-            // Enter Due Diligence period
-            const roundUpdate = [{
+            let maxGrantUSD = 0
+            let earmarkedUSD = 0
+            let fundingAvailableUSD = 0
+
+
+            let roundUpdate = [{
                 id: curRound['id'],
-                fields: {
-                    'Round State': RoundState.DueDiligence,
-                    'Proposals': allProposals.length,
-                    'OCEAN Price': tokenPrice,
-                    'Max Grant': maxGrantOCEAN,
-                    'Earmarked': earmarkedOCEAN,
-                    'Funding Available': fundingAvailableOCEAN,
-                }
+                fields: {}
             }]
+
+            switch(basisCurrency) {
+                case 'USD' :
+                maxGrantUSD = curRound.get('Max Grant USD')
+                earmarkedUSD = curRound.get('Earmarked USD')
+                fundingAvailableUSD = curRound.get('Funding Available USD')
+
+                maxGrant = maxGrantUSD/ tokenPrice
+                earmarked = earmarkedUSD / tokenPrice
+                fundingAvailable = fundingAvailableUSD / tokenPrice
+
+                roundUpdate = [{
+                    id: curRound['id'],
+                    fields: {
+                        'Round State': RoundState.DueDiligence,
+                        'Proposals': allProposals.length,
+                        'OCEAN Price': tokenPrice,
+                        'Max Grant': maxGrant,
+                        'Earmarked': earmarked,
+                        'Funding Available': fundingAvailable,
+                    }
+                }]
+                break
+
+                case 'OCEAN': 
+                const maxGrantOCEAN = curRound.get('Max Grant')
+                const earmarkedOCEAN = curRound.get('Earmarked')
+                const fundingAvailableOCEAN = curRound.get('Funding Available')
+
+                maxGrantUSD = maxGrantOCEAN / USDPrice
+                earmarkedUSD = earmarkedOCEAN / USDPrice
+                fundingAvailableUSD = fundingAvailableOCEAN / USDPrice
+
+                maxGrant = maxGrantOCEAN
+                earmarked = earmarkedOCEAN
+                fundingAvailable = fundingAvailableOCEAN 
+                roundUpdate = [{
+                    id: curRound['id'],
+                    fields: {
+                        'Round State': RoundState.DueDiligence,
+                        'Proposals': allProposals.length,
+                        'OCEAN Price': tokenPrice,
+                        'Max Grant': maxGrant,
+                        'Earmarked': earmarked,
+                        'Funding Available': fundingAvailable,
```

Comment:
I think the 3 fields from above(Max Grant, Earmarked, Funding Available) shouldn't be changed since their values are already in OCEAN and doesn't change.
We could also reuse some code, for example setting the(Round State, Proposals and OCEAN Price) before the switch statement since these are the same in both cases.